landing-page/content/common/spark-quickstart.md

```diff
@@ -0,0 +1,379 @@
+
+{{% codetabs "LaunchSparkClient" %}}
+{{% addtab "SparkSQL" checked %}}
+{{% addtab "SparkShell" %}}
+{{% addtab "PySpark" %}}
+{{% addtab "Notebook" %}}
+{{% tabcontent "SparkSQL"  %}}
+```sh
+docker exec -it spark-iceberg spark-sql
+```
+{{% /tabcontent %}}
+{{% tabcontent "SparkShell" %}}
+```sh
+docker exec -it spark-iceberg spark-shell
+```
+{{% /tabcontent %}}
+{{% tabcontent "PySpark" %}}
+```sh
+docker exec -it spark-iceberg pyspark
+```
+{{% /tabcontent %}}
+{{% tabcontent "Notebook" %}}
+```sh
+docker exec -it spark-iceberg notebook
+```
+{{< hint warning >}}
+The notebook server will be available at [http://localhost:8888](http://localhost:8888)
+{{< /hint >}}
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
+
+### Creating a table
+
+To create your first Iceberg table in Spark, run a [`CREATE TABLE`](../spark-ddl#create-table) command. In the following example, we'll create a table
+using `prod.nyc.taxis` where `prod` is the catalog name, `nyc` is the schema name, and `taxis` is the table name.
+
+
+{{% codetabs "CreateATable" %}}
+{{% addtab "SparkSQL" checked %}}
+{{% addtab "SparkShell" %}}
+{{% addtab "PySpark" %}}
+{{% tabcontent "SparkSQL"  %}}
+```sql
+CREATE TABLE prod.nyc.taxis
+(
+  VendorID bigint,
+  TripID bigint,
+  Trip_distance float,
+  Fare_amount double,
+  Store_and_fwd_flag string
+)
+PARTITIONED BY (VendorID);
+```
+{{% /tabcontent %}}
+{{% tabcontent "SparkShell" %}}
+```scala
+import org.apache.spark.sql.types.{DoubleType, FloatType, LongType, StringType, StructField, StructType}
+import org.apache.spark.sql.Row
+val schema = StructType( Array(
+    StructField("VendorID", LongType,true),
+    StructField("TripID", LongType,true),
+    StructField("Trip_distance", FloatType,true),
+    StructField("Fare_amount", DoubleType,true),
+    StructField("Store_and_fwd_flag", StringType,true)
+))
+val df = spark.createDataFrame(spark.sparkContext.emptyRDD[Row],schema)
+df.writeTo("prod.nyc.taxis").create()
+```
+{{% /tabcontent %}}
+{{% tabcontent "PySpark" %}}
+```py
+from pyspark.sql.types import DoubleType, FloatType, LongType, StructType,StructField, StringType
+schema = StructType([
+  StructField("VendorID", LongType(), True),
+  StructField("TripID", LongType(), True),
+  StructField("Trip_distance", FloatType(), True),
+  StructField("Fare_amount', DoubleType(), True),
+  StructField("Store_and_fwd_flag', StringType(), True)
+])
+
+df = spark.createDataFrame([], schema)
+df.writeTo("prod.nyc.taxis").create()
+```
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
+Iceberg catalogs support the full range of SQL DDL commands, including:
+
+* [`CREATE TABLE ... PARTITIONED BY`](../spark-ddl#create-table)
+* [`CREATE TABLE ... AS SELECT`](../spark-ddl#create-table--as-select)
+* [`ALTER TABLE`](../spark-ddl#alter-table)
+* [`DROP TABLE`](../spark-ddl#drop-table)
+
+### Adding data into a table
+
+Once your table is created, insert data using [`INSERT INTO`](../spark-writes#insert-into):
```

Comment:
Might want to make this generic if people are selecting PySpark or Scala dataframes.

Reply:
Ah right, fixed!